examples/pytorch/language-modeling/run_clm_no_trainer.py

```diff
@@ -208,7 +225,11 @@ def main():
     args = parse_args()
 
     # Initialize the accelerator. We will let the accelerator handle device placement for us in this example.
-    accelerator = Accelerator()
+    # If we're using tracking, we also need to initialize it here and it will pick up all supported trackers in the environment
+    if args.with_tracking:
+        accelerator = Accelerator(log_with="all")
+    else:
+        accelerator = Accelerator()
```

Comment:
```suggestion
    accelerator = Accelerator(log_with="all") if args.with_tracking else Accelerator()
```
Short enough to be in one line I think.